docs/features/key_cancellation.md

```diff
@@ -0,0 +1,50 @@
+# Key Cancellation
+
+Upon a selected key press down, if another key is pressed, it will release another key until it is physically pressed again.
+
+## How do I enable key cancellation {#how-do-i-enable-key-cancellation}
+
+in `rules.mk` add this:
+```make
+KEY_CANCELLATION_ENABLE = yes
+```
+
+By default, key cancellation is disabled even after adding to `rules.mk`. To enable it, you need to use the `KX_CATG` or `KX_CAON` keycode to enable it. The status is stored in persistent memory, so you shouldn't need to enable it again.
+
```

Comment:
````suggestion

Your `keymap.c` will then need a Key Cancellation definition:

```c
const key_cancellation_t PROGMEM key_cancellation_list[] = {
    // on key down
    //       |    key to be released
    //       |     |
    [0] = {KC_D, KC_A},
    [1] = {KC_A, KC_D},
    [2] = {KC_A, KC_F},
};

## Examples

### something something ima gamer
```c
const key_cancellation_t PROGMEM key_cancellation_list[] = {
    [0] = {KC_D, KC_A},
    [1] = {KC_A, KC_D},
};
```

````